packages/loader/src/gltf/parser/GLTFParser.ts

```diff
@@ -23,7 +23,7 @@ export abstract class GLTFParser {
     context: GLTFParserContext,
     ownerSchema: GLTFExtensionOwnerSchema,
     ...extra
-  ): EngineObject | void | Promise<EngineObject | void> {
+  ): Uint8Array | EngineObject | void | Promise<EngineObject | Uint8Array | void> {
```

Comment:
`EngineObject  | Promise<EngineObject | Uint8Array> ` is enough

And clear following code type with:
```tsresource = GLTFParser._createAndParse(extensionName, context, extensionSchema, ownerSchema, ...extra);```

Reply:
I think use `T|Promise<T>`  to refactor all plugin is better,  just similar to `context.get<T>()` , can open another PR and give it to me.